.pylintrc

```diff
@@ -0,0 +1,511 @@
+[MASTER]
```

Comment:
Did you change this at all from the original? bc we need to propose new ML rules. Also the decisions need to be properly motivated somewhere. For now just in the ReadMe. Also need to implement diff ways to display code quality info, maybe the report.txt is enough tho idk. This is from the assignment:

 The project implements different ways to display code quality information, considers multiple linters, critically analyses linter rules, and proposes new missing ML rules